subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/notes/tasks/FetchReleaseNotes.java

```diff
@@ -28,4 +39,9 @@ public void fetchReleaseNotes(FetchReleaseNotesTask task) {
         final String serializedData = releaseNotesSerializer.serialize(releaseNotes);
         IOUtil.writeFile(task.getOutputFile(), serializedData);
     }
+
+    private String buildUnknownRevisionMessage(RevisionNotFoundException exception) {
+        return "Version " + exception.getRevision() + " has not been found in VCS. Probably there is no" +
```

Comment:
This is an example of great exception message. Thank you!

One little suggestion I have is to make this more actionable. Can we suggest the user to fix the 'previousVersion' in the file or to create missing tag?